tests/js/server/replication2/replication2-replicated-state-document-store.js

```diff
@@ -35,6 +35,7 @@ const lp = require("@arangodb/testutils/replicated-logs-predicates");
 const lhttp = require('@arangodb/testutils/replicated-logs-http-helper');
 const dh = require("@arangodb/testutils/document-state-helper");
 const ch = require("@arangodb/testutils/collection-groups-helper");
+const serverHelper = require('@arangodb/test-helper');
```

Comment:
This is already included in line 32 as `helper`.

Reply:
Wasn't needed anyhow, erased